tests/base_gmsh/test_base_case.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+from pathlib import Path
+import hashlib
+import pytest
+import warnings
+
+# Different variations of gmsh come up with different meshes from the geo file.
+# If we see a mesh different to what we're expecting, warn the user and set
+# the tests to xfail
+b = Path(__file__).parent.resolve()
+expected_md5 = "41b8157a5e18f467dbd1e7538d1236d6"
+with open(b / "square.msh", "r") as f:
+    mesh_md5 = h = hashlib.md5(f.read().encode()).hexdigest()
```

Comment:
Somethingsomething hash collisions :grin: MD5 is probably fine for this anyway!

Reply:
Yeah, I don't think we need to be cryptographically secure here.

Reply:
Concern is more that MD5 doesn’t get deprecated out, but I don’t imagine that being an issue either.